Lib/xml/sax/handler.py

```diff
@@ -340,3 +341,57 @@ def resolveEntity(self, publicId, systemId):
                   property_xml_string,
                   property_encoding,
                   property_interning_dict]
+
+
+class LexicalHandler:
+    """Optional SAX2 handler for lexical events.
+
+    This handler is used to obtain lexical information about an XML
+    document, that is, information about how the document was encoded
+    (as opposed to what it contains, which is reported to the
+    ContentHandler), such as comments and CDATA marked section
+    boundaries.
+
+    To set the LexicalHandler of an XMLReader, use the setProperty
+    method with the property identifier
+    'http://xml.org/sax/handlers/LexicalHandler'."""
+
+    def xmlDecl(self, version, encoding, standalone):
```

Comment:
This method is not part of the `LexicalHandler` interface.
http://www.saxproject.org/apidoc/org/xml/sax/ext/LexicalHandler.html